nautobot/docs/release-notes/version-1.2.md

```diff
@@ -165,6 +165,16 @@ Just as with the UI, the `slug` can still always be explicitly set if desired.
 
 - [#650](https://github.com/nautobot/nautobot/issues/650) - Job Results UI now render job log messages immediately
 
+### Security
```

Comment:
To be consistent with our other release-notes, `Security` should come after `Fixed`.

Reply:
@jathanism If I'm reading this correctly it's adding the Security notes into the 1.2.2 section and not the 1.2.3 section?

Reply:
Good catch!

Reply:
Uh yeah I put it in the wrong section entirely. Whoops.